pages/index.js

```diff
@@ -11,11 +11,13 @@ export default class Home extends Component {
         <Helmet
           title={config.siteTitle}
         />
-        <h1>
-          Home
+        <h1 className="page--title">
+          International Relations Council
         </h1>
+        <h3 className="page--description">
```

Comment:
This is technically part of the page title, so it should be in the page__title class